cartographer/mapping/3d/submap_3d.cc

```diff
@@ -196,12 +197,20 @@ proto::SubmapsOptions3D CreateSubmapsOptions3D(
 }
 
 Submap3D::Submap3D(const float high_resolution, const float low_resolution,
-                   const transform::Rigid3d& local_submap_pose)
+                   const transform::Rigid3d& local_submap_pose,
+                   const int rotational_scan_matcher_histogram_size)
+    : Submap3D(high_resolution, low_resolution, local_submap_pose,
+               Eigen::VectorXf::Zero(rotational_scan_matcher_histogram_size)) {}
```

Comment:
No need for this convenience constructor. Call it like this at the call sites of this constructor instead?